.vscode/settings.json

```diff
@@ -1,7 +1,6 @@
 {
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "prettier.prettierPath": "./node_modules/prettier",
-  "editor.formatOnSave": true,
```

Comment:
it's good to keep `formatOnSave` ,

my prev comment was about `autoSave` instead